client/components/Header.js

```diff
@@ -1,3 +1,4 @@
+import jquery from 'jquery'
```

Comment:
Meh. 😒 

On the other hand: the `bundle.js` is 11.67MB already, jquery doesn't add that much more…